Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -423,7 +424,16 @@ - (void)_installScreenTimeWebpageControllerIfNeeded
     if (!PAL::isScreenTimeFrameworkAvailable())
         return;
 
-    if (!_page->preferences().screenTimeEnabled() || !_page->mainFrame()->url().protocolIsInHTTPFamily())
+    if (!_page->preferences().screenTimeEnabled() || !_page->mainFrame() || _page->mainFrame()->url().isNull() || !_page->mainFrame()->url().protocolIsInHTTPFamily())
+        return;
+
+    if (!_screenTimeConfigurationObserver) {
+        _screenTimeConfigurationObserver = adoptNS([PAL::allocSTScreenTimeConfigurationObserverInstance() initWithUpdateQueue:globalDispatchQueueSingleton(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0)]);
+        [_screenTimeConfigurationObserver addObserver:self forKeyPath:@"configuration.enforcesChildRestrictions" options:(NSKeyValueObservingOptionOld | NSKeyValueObservingOptionNew) context:&screenTimeConfigurationObserverKVOContext];
```

Comment:
You do not appear to use either old nor new values from the observation dict so I think you can get rid of those